src/Driver/IBMDB2/Result.php

```diff
@@ -28,6 +28,7 @@ public function __construct(private readonly mixed $statement)
 
     public function fetchNumeric(): array|false
     {
+        /** @var non-empty-list<mixed>|false $row */
         $row = @db2_fetch_array($this->statement);
```

Comment:
Can we contribute this upstream to the function stubs of PHPStan and Psalm? Overriding all those return types doesn't feel right.

Reply:
Yes, thank you!